### DISCUSSION TITLE
StoryManager 1.0.7

### DISCUSSION THREAD

**Discussion by Videogamers0** [Videogamers0/StoryManager#19] [Announcements]
Version 1.0.7

<hr /><em>This discussion was created from the release <a href='https://github.com/Videogamers0/StoryManager/releases/tag/v1.0.7'>StoryManager 1.0.7</a>.</em>

**Comment by glbetts** [comment]
I see that you've indicated updating the program to handle the new UI.  I've downloaded the executable but get the following error message when attempting to download an author's collection.
![Screenshot 2024-04-26 132154](https://github.com/Videogamers0/StoryManager/assets/33579321/1bb4fdd5-dcbd-40f8-b05d-d5bb1e81a2dc)

**Reply by Videogamers0** [reply]
The image you linked appears to be pasting an author's URL in the "Download story from URL:" section instead of the "Download stories from author:" section.

**Comment by glbetts** [comment]
Yes, the clipboard string.  I copy it from the authors ‘works’ page and paste it into ‘Download stories from author’ field. I’ve just tried again, using the author URL of :  https://www.literotica.com/authors/Chorisero/works/stories

 

I pasted it into the Download stories from author: field 



 

 

I then clicked on the Retrieve stories by Author button. And got this error4 message:

 



 

 

Somehow it appears to be adding the string, ‘https://www.literotic.com/authors/ before the URL I pasted in to that field.’

 

BTW:  thank you for responding.  This has been driving me nuts for a couple of days now.  I Love this program.

 

George …

**Comment by Videogamers0** [comment]
Oops, yeah there's a typo when parsing the URL. Fixed now 0fefe783556b16bd79807e9c891311de7f3f0e5c in version 1.0.8

**Comment by glbetts** [comment]
That fixed it!  Thank you.

 

George …
